src/patternfly/components/Nav/examples/Nav.md

```diff
@@ -515,23 +491,23 @@ The navigation system relies on several different sub-components:
 | `aria-current="page"` | `.pf-c-nav__link` |  Indicates the current page link. Can only occur once on page. |
 
 ### Usage
+
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-c-nav` | `<nav>` | Initiates a primary nav element. |
 | `.pf-c-nav__subnav` | `<section>` | Initiates a subnav section. |
-| `.pf-c-nav__list` | `<ul>` | Initiates default nav list. |
-| `.pf-c-nav__simple-list` | `<ul>` | Initiates simple nav list. |
-| `.pf-c-nav__item` | `<li>` | Initiates default nav list item. |
-| `.pf-c-nav__scroll-button` | `<button>` | Intitiates a nav scroll button. **Required for horizontal navs** |
-| `.pf-c-nav__link` | `<a>` | Initiates default nav list link. |
+| `.pf-c-nav__list` | `<ul>` | Initiates nav list. |
+| `.pf-c-nav__item` | `<li>` | Initiates nav list item. |
+| `.pf-c-nav__link` | `<a>` | Initiates nav list link. |
 | `.pf-c-nav__section` | `<section>` | Initiates a nav section element. |
 | `.pf-c-nav__section-title` | `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, `<h6>` | Initiates a nav section title. |
 | `.pf-c-nav__toggle` | `<span>` | Initiates a chevron indicating expandability of a `pf-c-nav__list-link`. |
 | `.pf-c-nav__toggle-icon` | `<span>` | Initiates a nav toggle icon wrapper. |
+| `.pf-c-nav__scroll-button` | `<button>` | Initiates a nav scroll button. **Required for horizontal navs** |
+| `.pf-m-horizontal` | `.pf-c-nav` | Modifies nav for the horizontal variation. |
+| `.pf-m-tertiary` | `.pf-c-nav` | Modifies nav for the tertiary variation. |
+| `.pf-m-light` | `.pf-c-nav` | Modifies nav for the light variation. **Note: for use with vertical nav, `.pf-m-light` is required on the page component's sidebar element (`.pf-c-page__sidebar`)**. |
```

Comment:
Should we change this back to what it was since it doesn't apply to horizontal nav?

Reply:
Yes!